src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -279,12 +258,10 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
                 refCreator.apply(variant, intervalsToWrite);
             }
         } catch (IOException ioe) {
-            throw new GATKException("Error writing PET", ioe);
+            throw new GATKException("Error writing reference ranges", ioe);
```

Comment:
and here, are we then assuming enableReferenceRanges is true?

Reply:
Good catch, it looks like there should be an `if` here like the one for vet on line 250.